yokai/messenger-bundle/1.0/config/packages/yokai_messenger.yaml

```diff
@@ -0,0 +1,46 @@
+yokai_messenger:
+    channels:
+        # Each channel has its own global configuration
+        swiftmailer:
```

Comment:
Is this the same as default configuration?

Reply:
The whole file is not the default configuration : because of the example, but yes, channels are all disabled by default (usage of `canBeEnabled` config helper)